### PR TITLE
Removed validation of empty attribute value

### LINKED ISSUE #178: Cosmwasm Std Empty String Attribute Causes Failure
If you have an attribute with an empty string as a value, when returning a response, it fails to send the response.

i.e. an execute msg that returns:
`Ok(Response::<Empty>::new().add_attribute("oopa", ""))`
will fail.

I am testing this using `cw-multi-test`
```
"Error executing WasmMsg:\n  sender: seda190vqdjtlpcq27xslcveglfmr4ynfwg7gmw86cnun4acakxrdd6gqcvkq45\n  Execute { contract_addr: \"contract0\", msg: {\"register_and_stake\":{\"public_key\":[2,116,118,117,180,232,142,56,102,99,116,236,213,232,115,227,219,19,252,0,182,237,97,27,66,160,175,139,129,164,203,169,134],\"proof\":[2,201,238,162,70,3,146,233,143,208,28,179,229,228,129,161,134,245,134,69,219,121,171,179,194,112,133,26,211,236,163,201,253,234,54,241,6,21,194,4,119,246,65,137,87,185,44,146,188,72,205,104,135,72,53,180,72,97,114,143,250,90,244,46,89,14,154,66,253,16,106,148,170,144,253,236,101,236,13,1,60],\"memo\":null}}, funds: [Coin { 1 \"aseda\" }] }",
```

Versions:
```
cosmwasm-std = { version = "1.5.5" }
cw-multi-test = "1.1.0"

Rust: rustc 1.77.2 (25ef9e3d8 2024-04-09)
```

This may be fixed on newer versions, or maybe this is caused by cw-multi?

### PR DESCRIPTION
**Pull request by DariuszDepta**
closes #178

### CODE REVIEW

File: src/wasm.rs [hunk 1]
```diff
@@ -357,24 +357,22 @@ impl<ExecC, QueryC> WasmKeeper<ExecC, QueryC> {
             .ok_or_else(|| Error::unregistered_code_id(code_id))?)
     }
 
+    /// Validates all attributes.
+    ///
+    /// In `wasmd`, before version v0.45.0 empty attribute values were not allowed.
+    /// Since `wasmd` v0.45.0 empty attribute values are allowed,
+    /// so the value is not validated anymore.
```
Comment (chipshort) [inline src/wasm.rs:364]: Nice 👍

**Review by chipshort — COMMENTED** [review]
Looks good, except for the breaking removal.

File: src/error.rs [hunk 2]
```diff
@@ -11,10 +11,6 @@ pub enum Error {
     #[error("Empty attribute key. Value: {0}")]
     EmptyAttributeKey(String),
 
-    /// Error variant for reporting an empty attribute value.
-    #[error("Empty attribute value. Key: {0}")]
-    EmptyAttributeValue(String),
```
[the remark this answers is not in the document]
Comment (DariuszDepta) [reply src/error.rs]: You are right, I have just deprecated this error variant.

**Review by chipshort — APPROVED** [review]
LGTM

**Pull request merged**